### ISSUE TITLE
New SQL Query: Luminosity contrast ratio is 2.6:1 which is less than 3:1 for "Close(X)" icon button of 'Query 1' tab control.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1043]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1241080

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section and select any of the item then some new menu item appears in the top like New SQL Query, New Stored Procedure..
Navigate to "New SQL Query" menu item and select it.
Navigate through all the controls under Query 1.
Navigate to "Close(X)" button of Query 1 tab control.
Open 'Color Contrast Analyser' tool and verify the color contrast of control.
Verify the issue.

Actual Result:
Luminosity contrast ratio is 2.6:1 which is less than 3:1 for "Close(X)" icon button of 'Query 1' tab control.

Expected Result:
Luminosity contrast ratio should be greater than or equal to 3:1 for "Close(X)" icon button of 'Query 1' tab control.


Note:
Similar issue exits for 'New Stored procedure' tab control.
This issue exists for all similar close buttons under 'Data Explorer' page.
Similar issue also exists for 'Close' buttons of 'New shell', 'New Query 1' under 'Data explorer' screen for created "Azure Cosmos DB for MongoDB API".

![image](https://user-images.githubusercontent.com/8519407/132717928-929853d2-fe22-4d9e-8797-526fb7047557.png)
